packages/newsroom-signup/src/actionCreators.ts

```diff
@@ -0,0 +1,269 @@
+import { AnyAction } from "redux";
```

Comment:
Note to us that there's now probably a lot of cruft in here and store in general that can be removed, though some may be needed for later steps so we can hold on for now.